Review part 1 (#13)
worldbank/dime-github-trainings: GitHub-trainings/github-pr-training/latex-slides/GitHub-PR-training.tex

		\column{.35\textwidth} % Left column and width
		\begin{itemize}
			\setlength\itemsep{.5em}

> 			\item Issues Area a great way to document what tasks need to be done

Reviewer: ```suggestion
			\item Issues are a great way to document what tasks need to be done
```

	\end{columns}
\end{frame}

\begin{frame}
	\frametitle{Create a PR}
	\begin{columns}[c]
		
		\column{.53\textwidth} % Right column and width
		\vspace{-.5cm}
		\begin{figure}
			\centering
			\includegraphics[width=\textwidth]{./img/create-pr-0.png}
		\end{figure}
		
		\column{.47\textwidth} % Left column and width
		Recap of PR creation best practices:
		\vspace{.5cm}
		\begin{itemize}
			\setlength\itemsep{.5em}

> 			\item Go to the \textit{Pull request} tab

Reviewer: ```suggestion
			\item Go to the \textit{``Pull request''} tab
```

		\column{.35\textwidth} % Left column and width
		\begin{itemize}
			\setlength\itemsep{2em}
			\item When reviewing a PR, always start by reading what the author or anyone else have written in the \textit{``Conversation"} tab

> 			\item Then go to the \textit{``Files changed"} tab - this is where you will review the new content of the files

Reviewer: ```suggestion
			\item Then go to the \textit{``Files changed''} tab - this is where you will review the new content of the files
```

		\column{.4\textwidth} % Left column and width
		\begin{itemize}
			\setlength\itemsep{1em}

> 			\item Make overall comments and comments that cannot be done to a specific section of the code in thread in the \textit{``Conversation"} tab

Reviewer: ```suggestion
			\item Make overall comments and comments that cannot be done to a specific section of the code in thread in the \textit{``Conversation''} tab
```

		\begin{itemize}
			\setlength\itemsep{1em}
			\item Start by going over the suggestions as they tend to be quick  

> 			\item Suggestions are ordered chronologically in the \textit{``Conversation"} tab and by line number in the \textit{``File changes"} tab

Reviewer: ```suggestion
			\item Suggestions are ordered chronologically in the \textit{``Conversation''} tab and by line number in the \textit{``File changes''} tab
```
